tests/playwright/templates/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Playwright Component Testing Template</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Titillium+Web:wght@300;400;700&display=swap"
```

Comment:
Thanks for adding the font 👍🏻 .

I couldn't figure out where the styles come from, so this may not be the right place for the question, but it's important:

Do you also load the styles from `src/index.scss` / `react-ui.css`? Now it may not make any difference in the snapshots, but we will see it later when we test more complicated components that rely on inheritance from foundation CSS.

Reply:
`tests/playwright/templates/index.jsx` is the second entry point that imports styles.